### PR TITLE
Skip refresh config on builds

### PR DESCRIPTION
**Pull request by nikhilgirraj**
The CI deployment fails due to failed integration tests.

### CODE REVIEW

**Comment by nikhilgirraj** [comment]
I like the idea of polling for a certain time after config refresh @tabman83.

**Comment by nikhilgirraj** [comment]
Tested with a release on CI. ✔️

**Comment by nikhilgirraj** [comment]
On second thoughts about Polling till the refresh is complete - wouldn't the other tests still fail since they might be started in parallel? @tabman83

**Comment by tabman83** [comment]
> On second thoughts about Polling till the refresh is complete - wouldn't the other tests still fail since they might be started in parallel? @tabman83

You can use collections, see https://xunit.net/docs/running-tests-in-parallel.html

**Comment by mszymeshop** [comment]
I have doubts regarding this, maybe I need a bit of explanation. Should we control the parallel run of the tests instead, as per @tabman83 link?

**Comment by nikhilgirraj** [comment]
@mszymeshop I checked, the tests do not run in parallel so far. See [src/Tests/CaptainHook.Api.Tests/config/ApiClientCollection.cs](https://github.com/eShopWorld/captain-hook/blob/master/src/Tests/CaptainHook.Api.Tests/config/ApiClientCollection.cs).

I have added a Polly retry for when there a 409 conflict in tests. @tabman83

File: src/Tests/CaptainHook.Api.Tests/Api/ControllerTestBase.cs [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
+        protected readonly AsyncRetryPolicy<HttpOperationResponse> ConflictRetryPolicy;
 
         public ControllerTestBase(ApiClientFixture testFixture)
         {
             AuthenticatedClient = testFixture.GetApiClient();
             UnauthenticatedClient = testFixture.GetApiUnauthenticatedClient();
+
+            ConflictRetryPolicy = Policy /* poll until no conflict */
+                .HandleResult<HttpOperationResponse>(msg => msg.Response.StatusCode == HttpStatusCode.Conflict)
+                .WaitAndRetryAsync(30, i => TimeSpan.FromMilliseconds(2000));
```
Comment (tabman83) [inline src/Tests/CaptainHook.Api.Tests/Api/ControllerTestBase.cs:24]: FromSeconds(2) is more readable.

Maybe we can even check every 5 sec
CC @mszymeshop

**Comment by nikhilgirraj** [comment]
Reverted the changes for now to make the CI green.

**Pull request merged**